tests/CMakeLists.txt

```diff
@@ -46,9 +46,9 @@ add_test(NAME nodeos_sanity_test COMMAND tests/nodeos_run_test.py -v --sanity-te
 add_test(NAME nodeos_run_test COMMAND tests/nodeos_run_test.py -v --clean-run --dump-error-detail WORKING_DIRECTORY ${CMAKE_BINARY_DIR})
 
 add_test(NAME p2p_dawn515_test COMMAND tests/p2p_tests/dawn_515/test.sh WORKING_DIRECTORY ${CMAKE_BINARY_DIR})
-#if(BUILD_MONGO_DB_PLUGIN)
-#   add_test(NAME nodeos_run_test-mongodb COMMAND tests/nodeos_run_test.py --mongodb -v --dump-error-detail WORKING_DIRECTORY ${CMAKE_BINARY_DIR})
-#endif()
+if(BUILD_MONGO_DB_PLUGIN)
+  add_test(NAME nodeos_run_test-mongodb COMMAND tests/nodeos_run_test.py --mongodb -v --clean-run --dump-error-detail WORKING_DIRECTORY ${CMAKE_BINARY_DIR})
+endif()
```

Comment:
I think we should make it one of the long-running tests.

Reply:
Disagree, Its not much more than the nodeos smoke test. Here are the numbers from the Darwin build:

27/38 Test #27: unit_test_wavm .......................   Passed  425.79 sec
31/38 Test #31: nodeos_run_test ......................   Passed   59.68 sec
33/38 Test #33: nodeos_run_test-mongodb ..............   Passed   94.07 sec
35/38 Test #35: restart-scenarios-test-resync ........   Passed  153.00 sec